src/AutoBuild.jl

```diff
@@ -1276,15 +1290,36 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
     open(joinpath(code_dir, "README.md"), "w") do io
         print(io,
               """
-              # $(src_name)_jll.jl
+              # $(src_name)_jll.jl `v$(build_version)`
 
-              This is an autogenerated package constructed using [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl).
-
-              For more details about JLL packages, see `BinaryBuilder.jl` [documentation](https://juliapackaging.github.io/BinaryBuilder.jl/dev/jll/).
-
-              """,
-              iszero(length(keys(products_info))) ? "" :
+              This is an autogenerated package constructed using [`BinaryBuilder.jl`](https://github.com/JuliaPackaging/BinaryBuilder.jl).""")
+        if from_yggdrasil
+            print(io, " The recipe can be found on [`Yggdrasil`](https://github.com/JuliaPackaging/Yggdrasil/).")
+        end
+        println(io)
+        println(io)
+        print(io,
               """
+              For more details about JLL packages and how to use them, see `BinaryBuilder.jl` [documentation](https://juliapackaging.github.io/BinaryBuilder.jl/dev/jll/).
+
+              The tarballs have been built""")
+        if length(sources) > 0
+            println(io, " from these sources:")
+            println(io)
+            print_source.(Ref(io), sources)
+            println(io)
+        else
+            print(io, " ")
+        end
+        println(io, "for the following platforms:")
+        println(io)
+        for p in platforms
+            println(io, "* `", p, "` (`", triplet(p), "`)")
+        end
```

Comment:
I'm not sure I like the format of the README of https://github.com/giordano/libsodium_jll.jl: the sources and the platforms are verbosely listed in a bullet list, the products are in a new section, printed as Julia code.

It's probably worth to add the dependencies as well

Reply:
I've moved each piece of information to its own section, I think it's easier to parse in this way.  I've added the dependencies as well and the products are in a bullet list